config/sync/block.block.healthconditionsatoz_2.yml

```diff
@@ -0,0 +1,28 @@
+uuid: 70b5dc81-8444-46e8-800a-19703a0ab734
+langcode: en
+status: true
+dependencies:
+  module:
+    - ctools
+    - nidirect_health_conditions
+  theme:
+    - nicsdru_nidirect_theme
+id: healthconditionsatoz_2
```

Comment:
Is there a way to make the `_2` suffix more descriptive? I can't work out what this block does that `block.block.healthconditionsatoz.yml` doesn't provide. If the original doesn't _quite_ do what's needed, maybe we can change it to avoid spawning a new block?

Reply:
the `_2` suffix is indicatative that the block appears in the second sidebar region, I can amend the machine name to something more meaningful if you would like. The block is in essence rendered contextually in two regions, if on the search page it (probably) will render on the first sidebar region, while if viewing a health condition page it will render on the second sidebar region. We will also need to amend the original block as well the link txt for resetting the form should only appear when viewing in on a search index page, if viewing on a health condition node page it should not be present

Reply:
Thanks, that explains it. Probably wise to give the filename/machine name a tweak or add a comment to indicate what you've outlined above, but if it's a massive PITA this shouldn't be a show stopper.

Reply:
I've updated the 2 health condition blocks tat are output on the 1st and 2nd sidebar regions so that the second sidebar block are suffixed with _sidebar_2 as the machine id